src/components/Card.js

```diff
@@ -0,0 +1,65 @@
+import { Component } from "react"
+import styled from "styled-components"
+
+const _CardContainer = styled.div.attrs({
+  className: "br4 ma1",
+})`
+  ${({ selected }) =>
+    selected &&
+    `border: thin solid var(--gray);	
+  `};
+`
+
+const _Card = styled.div.attrs({
+  className: "flex flex-column ma1 items-center br4",
+})`
+  background-color: var(--yellow);
+  ${({ width, height }) =>
+    `width: ${width};
+    height: ${height};	
+  `};
+`
+const _CardTitle = styled.p.attrs({
+  className: "font-4 mono mt6 mb2 b tc",
+})`
+  color: var(--gray);
+`
+const _CardDescription = styled.p.attrs({
+  className: "font-5 mono ma2 tc",
+})`
+  color: var(--gray);
+`
+const _CardImage = styled.img.attrs({
+  className: "ma2",
+})`
+  height: 40%;
+  width: 40%;
+`
+
+class Card extends Component {
+  onClick = () => {
```

Comment:
@abdhalees Is this being used?

Reply:
this adds the border when the card is selected. it will be the same in other Carousel views so no need to pass it from the parent component.